netci.groovy

```diff
@@ -55,15 +57,23 @@ def project = GithubProject
                 case 'OSX':
                     newJob.with{
                         steps{
-                            shell("./cibuild.sh --scope Test --target ${runtime}")
+                            buildCmd="./cibuild.sh --scope Test --target ${runtime}"
+                            if (runtime == "Mono") {
+                                buildCmd += " --host Mono"
```

Comment:
`--target` selects the configuration (like `Debug-MONO`) and `--host` selects coreclr/mono to use for running msbuild.